lib/index.js

```diff
@@ -77,11 +76,10 @@ class PWMetrics {
     return lighthouse(this.url, lhOpts, perfConfig)
       .then(res => metrics.prepareData(res))
       .then(data => {
-        if (this.flags.submit) {
-          return this.appendResults(data);
```

Comment:
@paulirish yeap, the reason I did it, to not spam sheets with each request (in your case it will be 100 requests). If you think it's ok to send data on each run, then we can bring it back.